dex/src/main/scala/com/wavesplatform/dex/actors/address/AddressActor.scala

```diff
@@ -375,26 +374,24 @@ class AddressActor(
       } else {
         log.debug(s"$command, to cancel: ${toCancelIds.mkString(", ")}")
         runWithIgnoredSpan {
-          context.actorOf(BatchOrderCancelActor.props(toCancelIds.toSet, command.source, self, sender(), settings.batchCancelTimeout))
+          context.actorOf(BatchOrderCancelActor.props(
+            toCancelIds.toList,
+            command.source,
+            self,
+            sender(),
+            settings.batchCancelTimeout
+          ))
         }
       }
 
     case command: Command.CancelOrders =>
-      val allActiveOrderIds = getActiveLimitOrders(None).map(_.order.id()).toSet
-      val toCancelIds = allActiveOrderIds.intersect(command.orderIds)
-      val unknownIds = command.orderIds -- allActiveOrderIds
-
-      log.debug(
```

Comment:
Let's log the command: `log.debug("{}", command)`

Reply:
added